apps/functions/README.md

```diff
@@ -0,0 +1,53 @@
+# Azure Functions
+
+This folder contains the code for the Back Office [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview?pivots=programming-language-javascript) (referred to herein as Functions).
+
+Functions are used to augment the functionality of the api and web apps, by providing (for example) a mechanism to respond to message queues, or trigger some background functionality. 
+
+## Dev Setup
+
+To be able to create new Functions, and run functions locally install azure-functions-core-tools:
+
+`npm install -g azure-functions-core-tools@4`
+
+Specific instructions for local dev are provided by each Function. In general, to start a function locally, use `func start`. 
+
+## Terminology
+
+The terminology around Functions can be confusing. Some key terms are:
+
+* Azure Functions - the overall solution in Azure for "serverless" code execution
+* Trigger - the cause/method by which a Function is started (e.g. HTTP call, or responding to an event)
+* Azure Function - a specific Function (code) that can be executed to do a job
+* Function Project - a container for one or more Functions. Functions in a project share configuration, but can each respond to a specific trigger.
+
+## New Function
+
+To create a new JavaScript Function, see the Azure documentation: 
+
+[Quickstart: Create a JavaScript function in Azure from the command line](https://learn.microsoft.com/en-gb/azure/azure-functions/create-first-function-cli-node?tabs=azure-cli%2Cbrowser&pivots=nodejs-model-v4)
+
+The key parts are:
+
+1. Ensure you have `azure-functions-core-tools` installed (try running `func --version` if you're not sure)
+2. Create a function project with `func init <project-name>`
+3. Navigate into the folder now created `cd <project-name>`
+4. Create a function from a template with `func new --template <template-name> --name <func-name>`
```

Comment:
Should we provide more guidance on the template to use?

Reply:
I couldn't find an obvious link with a list, which would be my ideal. We could mention a few that we're likely to use.